### ISSUE TITLE
3.0 - Improve Exports screen accessibility, markup

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#8112]
### Explain your enhancement (please be detailed)
The Downloads > Reports > Export screen should be updated to improve overall markup and accessibility.

### Justification or use case
Running an automated accessibility scan reveals multiple issues with the Exports screen:
- Form elements are missing labels (I think no inputs on this screen have labels)
- ID attributes are not unique
- Heading levels increase incorrectly (h1 > h3)

We started formalizing form/input markup with #7739 and can resolve many of the accessibility issues while updating the markup ([reference for markup](https://github.com/easydigitaldownloads/easy-digital-downloads/wiki/Form-Input-Markup)).

**Comment by LisaCee** [comment]
Since this is another big form markup issue, @spencerfinnell do you remember why it was decided that checkboxes and radio buttons don't have the `class="edd-form-group__label`?  [Here's](https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7739) where the majority of the discussion took place.  Is that something we should make note of on the [wiki](https://github.com/easydigitaldownloads/easy-digital-downloads/wiki/Form-Input-Markup) so that it is clear that it was intentional?

**Comment by spencerfinnell** [comment]
@LisaCee If I recall correctly it was because `.edd-form-group__label` referred to the _visual_ label for the group vs. the semantic label class. In the wiki examples the `.edd-form-group__label` usage in Radio and Text creates the same visual label hierarchy.

**Comment by robincornett** [comment]
That makes sense, although I didn't remember that when working on #7742 and so there are some checkbox labels on the download edit screen now which do have the class. I ended up adding the following CSS to compensate for what I considered to be unexpected behavior:

```css
.edd-form-group__input[type="checkbox"] + label,
.edd-form-group__input[type="radio"] + label {
	display: unset;
}
```
Which did cover my error, as it appears to be an error, but I think adding a note to the wiki would be helpful as I'm likely to still want to add a class to checkbox/radio labels, since the other labels have them.

**Comment by spencerfinnell** [comment]
@robincornett That is to help the labels appear inline/next to the checkbox/radio? I think that makes sense and is a good thing to define to help  address any extension CSS that might be over stepping accidentally.

**Comment by robincornett** [comment]
Yes, that is what it does--to fix my incorrect markup, as I was adding the `edd-form-group__label` to _all_ `label` elements, regardless of context. I do know I made a similar mistake in the git updater plugin which required similar intervention, so yes, there will be extensions needing it, sadly.

**Comment by LisaCee** [comment]
PR #8151
